Packs/AWS-EC2/Scripts/AwsEC2SyncAccounts/AwsEC2SyncAccounts.py

```diff
@@ -117,7 +117,8 @@ def update_ec2_instance(account_ids: list[str], ec2_instance_name: str) -> str:
 def main():
     try:
         args: dict = demisto.args()
-        account_ids, readable_output = get_account_ids(args.get('org_instance_name'))
+        account_ids, readable_output = get_account_ids(
+            args.get('org_instance_name'), arg_to_number(args.get('max_accounts')))
```

Comment:
arg_to_number will raise an error when the argument is None, when you get `args.get('max_accounts')` None form this argument you will get an error

Reply:
There's a default value for the argument